src/Models/NonhydrostaticModels/boundary_mass_fluxes.jl

```diff
@@ -149,8 +149,8 @@ function initialize_boundary_mass_fluxes(velocities::NamedTuple)
         total_area_matching_scheme_boundaries += boundary_fluxes.top_area
     end
 
-    boundary_fluxes = merge(boundary_fluxes, (; left_matching_scheme_boundaries,
-                                                right_matching_scheme_boundaries,
+    boundary_fluxes = merge(boundary_fluxes, (; left_matching_scheme_boundaries = Tuple(left_matching_scheme_boundaries),
+                                                right_matching_scheme_boundaries = Tuple(right_matching_scheme_boundaries),
                                                 total_area_matching_scheme_boundaries))
```

Comment:
What are the benefits of a NamedTuple vs type here

Reply:
I guess just the simplicity of using a common type that everyone's familiar with. I don't see disadvantages of using `NamedTuple`s.